vault-tool/src/Network/VaultTool/Types.hs

```diff
@@ -1,12 +1,49 @@
+{-# LANGUAGE GADTs #-}
 {-# LANGUAGE OverloadedStrings #-}
 
-module Network.VaultTool.Types where
+module Network.VaultTool.Types (
+    VaultAddress (..),
+    VaultAppRoleId (..),
+    VaultAppRoleSecretId (..),
+    VaultAppRoleSecretIdAccessor (..),
+    VaultAuthToken (..),
+    VaultConnection (..),
+    Authenticated,
+    Unauthenticated,
+    authenticatedVaultConnection,
+    unauthenticatedVaultConnection,
+    VaultException (..),
+    VaultMountedPath (..),
+    VaultSearchPath (..),
+    VaultSecretPath (..),
+    VaultUnsealKey (..),
+) where
 
 import Control.Exception (Exception)
 import Data.Aeson
 import Data.ByteString (ByteString)
 import Data.Text (Text)
 import qualified Data.ByteString.Lazy as BL
+import Network.HTTP.Client (Manager)
+
+-- |The APIs exported by this library expect a 'VaultConnection' which is used to know where a vault server is and how
```

Comment:
Please include a space after `|` here and below.

Reply:
2c42064fa561f294710ccd8b3ca290ace3d42d3f